_posts/2019-08-07-vim-basics.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: Vim Guide for Beginners
+date: 2019-08-07 10:28:02
+categories: ["learning", "vim"]
+author: bronzdoc
+---
+
+# VIM basics
```

Comment:
```suggestion
```

This title will appear in the index page, so let's remove it:
<img width="1046" alt="Screen Shot 2019-08-07 at 5 47 21 PM" src="https://user-images.githubusercontent.com/7442887/62656871-8c537700-b93b-11e9-8f9f-cd3dcaf51e84.png">